packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -266,17 +265,19 @@ class DatePicker extends React.Component {
     this.setState({
       selectedDate: date,
     });
-
     if (this.props.onChange) {
       this.props.onChange(event, date && date.isValid() ? date.format(DateUtil.ISO_EXTENDED_DATE_FORMAT) : '');
     }
   }
 
-  handleChangeRaw(event) {
+  handleChangeRaw(event, date) {
+    if (date) {
+      this.handleChange(DateUtil.createSafeDate(date), event);
+    }
     this.dateValue = event.target.value;
 
     if (this.props.onChangeRaw) {
-      this.props.onChangeRaw(event, event.target.value);
+      this.props.onChangeRaw(event);
```

Comment:
Breaking change?

Reply:
No, but it also is unnecessary, so I reverted it: [89c3693](https://github.com/cerner/terra-framework/pull/1004/commits/89c3693f31162adfc53f90f681cdfda21588b6b0)